docs/guides/local-testing-mode.ipynb

```diff
@@ -37,12 +39,36 @@
+    "Use local testing mode (available with `qiskit-ibm-runtime` v0.22.0 or later) to test programs before fine-tuning them and sending them to real quantum hardware.  After using local testing mode to verify your program, all you need to change is the backend name to run it on a QPU.\n",
     "\n",
     "To use local testing mode, specify one of the fake backends from ``qiskit_ibm_runtime.fake_provider`` or specify a Qiskit Aer backend when instantiating a Qiskit Runtime primitive or a session.\n",
     "\n",
     "- **Fake backends**: The [fake backends](/docs/api/qiskit-ibm-runtime/fake-provider)  in ``qiskit_ibm_runtime.fake_provider`` mimic the behaviors of IBM&reg; QPUs by using QPU snapshots. The QPU snapshots contain important information about the QPU, such as the coupling map, basis gates, and qubit properties, which are useful for testing the transpiler and performing noisy simulations of the QPU. The noise model from the snapshot is automatically applied during simulation.\n",
-    "- **Aer simulator**: Simulators from [Qiskit Aer](simulate-with-qiskit-aer) provide higher-performance simulation that can handle larger circuits and [custom noise models](/docs/guides/build-noise-models). It also supports Clifford simulation mode, which can efficiently simulate Clifford circuits with a large number of qubits.\n",
+    "- **Aer simulator**: Simulators from [Qiskit Aer](/docs/guides/simulate-with-qiskit-aer) provide higher-performance simulation that can handle larger circuits and [custom noise models](/docs/guides/build-noise-models). A list of simulation method options are available when you use `AerSimulator` in local testing mode. See the [Clifford simulation mode example](#clifford-sim), which demonstrates how to efficiently simulate Clifford circuits with a large number of qubits.\n",
+    "\n",
+    "    <details>\n",
+    "    <summary>**List of simulation methods available from Qiskit Aer**</summary>\n",
+    "\n",
+    "    See the [`AerSimulator`](https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator) documentation for more information.\n",
+    "\n",
+    "    * ``\"automatic\"``: Default simulation method. Select the simulation method automatically, based on the circuit and noise model.\n",
+    "\n",
+    "    * ``\"statevector\"``: A dense statevector simulation that can sample measurement outcomes from *ideal* circuits with all measurements at end of the circuit. For noisy simulations, each shot samples a randomly sampled noisy circuit from the noise model.\n",
+    "\n",
+    "    * ``\"density_matrix\"``: A dense density matrix simulation that can sample measurement outcomes from *noisy* circuits with all measurements at end of the circuit.\n",
```

Comment:
```suggestion
    "    * ``\"density_matrix\"``: A density matrix simulation that can sample measurement outcomes from *noisy* circuits with all measurements at the end of the circuit.\n",
```

Reply:
Is "dense density" correct? It sounds odd to me, but I found some other instances of it.